lib/phoenix/endpoint.ex

```diff
@@ -435,24 +435,15 @@ defmodule Phoenix.Endpoint do
 
   defp plug() do
     quote location: :keep do
-      @behaviour Plug
+      use Plug.Builder
```

Comment:
**Note**: by using Plug.Builder, the Plug.Conn functions are imported. If this is undesirable, we can `import Plug.Conn, only: []` after using Plug.Builder.